portfolio/src/main/webapp/script.js

```diff
@@ -54,22 +54,50 @@ function addRandomTVQuote() {
 }
 
 /**
- * Using async and await to fetch list data.
+ * Load comments from server and add them to page.
  */
-async function getDataWithAsyncAwait() {
-  const response = await fetch('/data');
-  const messages = await response.json();
+function loadComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
```

Comment:
nit: put brackets around the inputs for the arrow functions. 
This is not strictly necessary in this instance but we generally try and do it for consistency.

Reply:
Done. I think?